code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -212,41 +221,255 @@
+	if(prob(20*amphetamine_power))
 		M.emote(pick("twitch","drool","moan"))
 	..()
 
-/datum/reagent/drug/methamphetamine/addiction_act_stage4(mob/living/carbon/human/M)
+/datum/reagent/drug/amphetamine/addiction_act_stage4(mob/living/carbon/human/M)
 	if((M.mobility_flags & MOBILITY_MOVE) && !ismovableatom(M.loc))
 		for(var/i = 0, i < 8, i++)
 			step(M, pick(GLOB.cardinals))
-	M.Jitter(20)
-	M.Dizzy(20)
-	M.adjustToxLoss(5, 0)
-	if(prob(50))
+	M.Jitter(10*amphetamine_power)
+	M.Dizzy(15*amphetamine_power)
+	M.adjustToxLoss(2.5*amphetamine_power, 0)
+	if(prob(25*amphetamine_power))
 		M.emote(pick("twitch","drool","moan"))
 	..()
 	. = 1
 
+/datum/reagent/drug/amphetamine/methamphetamine
+	name = "Methamphetamine"
+	description = "Reduces stun times by about 300%, speeds the user up, and allows the user to quickly recover stamina while dealing a small amount of Brain damage."
+	reagent_state = LIQUID
+	color = "#FAFAFA"
+	overdose_threshold = 20
+	addiction_threshold = 10
+	metabolization_rate = 0.75 * REAGENTS_METABOLISM
+	amphetamine_power = 4
+	modifier = 10
+
+/datum/reagent/drug/amphetamine/adderal
+	name = "Adderal"
+	description = "Weak Amphetamine, low doses increase focus and efficiency, high doses result in addiction, brain damage, and organ failure."
+	reagent_state = LIQUID
+	color = "#FAFAFA"
+	overdose_threshold = 40
+	addiction_threshold = 20
+	metabolization_rate = 1 * REAGENTS_METABOLISM
+	amphetamine_power = 0.5
+	modifier = 0
+	var/adderal_point_gen = 10
+
+/datum/reagent/drug/amphetamine/adderal/on_mob_life(mob/living/M)
+	var/amphetamine_power_effect = volume * amphetamine_power
+	if(M.mind && (M.mind.assigned_role == "Scientist" || M.mind.assigned_role == "Research Director"))
+		if(SSresearch.science_tech)
+			if(amphetamine_power_effect > 4.7)
+				var/adderalpwr = 6.3 - amphetamine_power_effect //dose your amphetamine/adderal/meth well!
+				SSresearch.science_tech.add_point_list(list(TECHWEB_POINT_TYPE_GENERIC = adderal_point_gen * adderalpwr)) 	
+	..()
+
+/datum/reagent/drug/amphetamine/gojuice
+	name = "Go Juice"
+	description = "Fizzy and sweet juice. Commercially promoted to every security force. Is not considered a drug under space law , even though it is jam filled with amphetamines. Extremely addictive."
+	reagent_state = LIQUID
+	color = "#829929"
+	metabolization_rate = 0.5 * REAGENTS_METABOLISM
+	overdose_threshold = 5
+	addiction_threshold = 2.5
+	amphetamine_power = 6
+	modifier = 10
+
+/datum/reagent/drug/amphetamine/gojuice/on_mob_life(mob/living/M)
+	if(prob(25))
+		M.adjustOrganLoss(ORGAN_SLOT_HEART, pick(0.2, 0.4, 0.6, 0.8, 0.9, 1))
+
+/datum/reagent/drug/amphetamine/gojuice/overdose_process(mob/living/M)
+	if(prob(33))	
+		M.adjustOrganLoss(ORGAN_SLOT_HEART, pick(2, 2.4, 2.8, 3.2, 3.6, 4))
+	if(prob(33))	
+		M.adjustOrganLoss(ORGAN_SLOT_BRAIN, pick(2, 2.4, 2.8, 3.2, 3.6, 4))
+	if(prob(33))	
+		M.adjustOrganLoss(ORGAN_SLOT_LUNGS, pick(2, 2.4, 2.8, 3.2, 3.6, 4))
+	M.Jitter(10)
+	M.Dizzy(10)
```

Comment:
how does a 15u syringe full of this stuff work?

on top of addiction

Reply:
shit you are right. it would pretty much kill you.